tests/persistSync.test.tsx

```diff
@@ -0,0 +1,173 @@
+import create from '../src/index'
+import { persist } from '../src/middleware'
+
```

Comment:
maybe a little `describe` over this whole file could structure this a bit more?

Reply:
would you mind please check that @barbogast

Reply:
👍 will add